pom.xml

```diff
@@ -19,7 +19,17 @@
             </plugin>
         </plugins>
     </build>
-
+    <dependencyManagement>
+      <dependencies>
+        <dependency>
+          <groupId>org.springframework</groupId>
+          <artifactId>spring-framework-bom</artifactId>
+          <version>4.3.21.RELEASE</version>
+          <type>pom</type>
+          <scope>import</scope>
```

Comment:
I didn't even know about scope=import. Nice!

Reply:
Yeah, it's the method that Spring recommends to keep (transitive) dependency versions consistent: https://docs.spring.io/spring-framework/docs/4.3.x/spring-framework-reference/htmlsingle/#overview-maven-bom .